lib/rubocop/cop/rspec/subject_stub.rb

```diff
@@ -22,6 +25,19 @@ module RSpec
       #     end
       #   end
       #
+      #   # bad
+      #   describe Article do
+      #     subject(:foo) { Article.new }
+      #
+      #     context 'nested subject' do
+      #       subject(:article) { Article.new }
+      #
+      #       it 'indicates that the author is unknown' do
+      #         expect(article.description).to include('by an unknown author')
```

Comment:
It’s only a "bad" example if a method is stubbed on the "subject" object, right? E.g.
```suggestion
      #       it 'indicates that the author is unknown' do
      #         allow(article).to receive(:author).and_return(nil)
      #         expect(article.description).to include('by an unknown author')
```

Reply:
I've applied your comment.

> It’s only a "bad" example if a method is stubbed on the "subject" object, right? E.g.

I see. That's true.
I may have mixed up some of the specifications and problems.
But if that's the case, I'd like to make the following an offence as well.

```
allow(foo).to receive(:wow) # no offensive, but isn't this offensiv?
allow(bar).to receive(:wow) # offensive
```

Reply:
I'm out of context. What is `foo`?